include/tvm/relax/transform.h

```diff
@@ -89,6 +89,13 @@ TVM_DLL Pass ToNonDataflow();
  */
 TVM_DLL Pass CallTIRRewrite();
 
+/*!
+ * \brief Perform explicit tensor allocation for call_dps_packed.
+ *
+ * \return The Pass.
+ */
+TVM_DLL Pass CallDPSPackedRewrite();
```

Comment:
I’m wondering if we need this pass. I suppose that the lowering of `call_tir` and `call_dps_packed` are the same (the diff check  also suggests this). The reason we want to introduce two is to reduce confusion. In terms of their lowering, I suppose they can directly share a same logic, which makes things easy (say when someone wants to change CallTIRRewrite, it won’t need to change CallDPSPackedRewrite in the same way as well).

Reply:
It might be fine to make them syntactic sugar for the same underlying operator. There are only two differences I can think of:
1. `call_tir` is generally assumed to be pure, whereas `call_dps_packed` may not be (unless we want to require it)
2. `call_tir` expects a `PrimFunc` as its operand, whereas `call_dps_packed` expects a `PackedFunc`.

Reply:
Thanks for the comments! Actually I was struggling to add CallDPSPackedRewrite, it does duplicate lots of code from CallTIRRewrite, and modifying CallTIRRewrite to work for both call_tir and call_dps_packed brings some confusion (probably a different pass name). I removed CallDPSPackedRewrite in the latest commit, we can add it back when the differences are introduces.